gcp_variant_transforms/testing/integration/vcf_to_bq_tests/presubmit_tests/medium_tests/gnomad_genomes_GRCh37_chrX_head2500_run_vep.json

```diff
@@ -12,26 +8,25 @@
     "runner": "DataflowRunner",
     "run_annotation_pipeline": "True",
     "annotation_output_dir": "gs://integration_test_runs/temp/vep_output/{TABLE_NAME}",
-    "disk_size_gb": 150,
-    "num_workers": 1,
+    "num_workers": 2,
```

Comment:
Setting `num_workers` to 2 gives me 500 GB, which didn't throw any errors. It isn't clear to me, however, why that much space would be needed. Also, I successfully reran the `platinum_NA12877_hg38_10K_lines_run_vep.json` while maintaining the original `"num_workers": 1," line. So maybe something else happened during those last failures and dropping back down to 1 hear would be fine.